tests/xml2/XMLBearTest.py

```diff
@@ -79,6 +79,20 @@ def load_testdata(filename):
     settings={'xml_dtd': dtd_url},
     tempfile_kwargs={'suffix': '.xml'})
 
+XMLBearStyleTest = verify_local_bear(
+    XMLBear,
+    valid_files=(valid_xml_file,),
+    invalid_files=(invalid_xml_chars,),
+    settings={'xml_style': 'oldxml10'},
```

Comment:
This style is not being tested.  This test class is identical to `XMLBearCorrectedTest` , etc.
You need to find an XML style issue, and ensure that the linter bear reports the error that `xmllint` emits and that its severity is correct.